superset/assets/spec/javascripts/explore/utils_spec.jsx

```diff
@@ -12,48 +13,84 @@ describe('utils', () => {
     expect(uri1.toString()).to.equal(uri2.toString());
   }
 
-  it('getExploreUrl generates proper base url', () => {
+  it('createExplore generates proper base url', () => {
```

Comment:
prob outside the scope of this PR but just a thought for test organization, if you break out a `describe('specific utils func...', () => { ... })` you wouldn't have to repeat the name for each test and could say `expect('it ....')`.

Reply:
fixed.